### PR TITLE
Add mkdir step in dockerfile

### PR DESCRIPTION
**Pull request by ebezzi**
#### Reviewers
**Functional:** 
@atolopko-czi 

**Readability:** 

---

## Changes
- add
- remove
- modify

### CODE REVIEW

**Review by atolopko-czi — APPROVED** [review]

File: hosted/Dockerfile [hunk 1]
```diff
@@ -29,6 +29,8 @@ RUN python3 create_rdev_config.py
 
 RUN make build-client
 
+RUN mkdir -p /server/app/../common/web/templates/
```
Comment (atolopko-czi) [inline hosted/Dockerfile:32]: or just `/server/common/web/templates/`

Comment (ebezzi) [reply hosted/Dockerfile:32]:
```suggestion
RUN mkdir -p /server/common/web/templates/
```

**Review by atolopko-czi — CHANGES_REQUESTED** [review]
test didn't work; trying a few things; hold off on merge

**Review by atolopko-czi — APPROVED** [review]
Tested, works!

**Pull request merged**